### PR TITLE
SPOI-12088 Added the redshift jdbc driver licenses in app templates

### PR DESCRIPTION
**Pull request by chaithu14**
@ashwinchandrap Please review and merge.

### CODE REVIEW

File: app-templates/kinesis-to-redshift/LICENSE [hunk 1]
```diff
@@ -0,0 +1,215 @@
+The Redshift JDBC driver is subject to the following licenses:
```
Comment (ashwinchandrap) [inline app-templates/kinesis-to-redshift/LICENSE]: Can you mention the exact java class with package path that uses this driver. Probably the redshift output operator.

File: app-templates/s3-to-redshift/LICENSE [hunk 2]
```diff
@@ -0,0 +1,215 @@
+The Redshift JDBC driver is subject to the following licenses:
```
Comment (ashwinchandrap) [inline app-templates/s3-to-redshift/LICENSE]: Can you mention the exact java class with package path that uses this driver. Probably the redshift output operator.

**Comment by chaithu14** [comment]
@ashwinchandrap Added java class with classpath in LICENSE file. Please review.

**Comment by yogidevendra** [comment]
LGTM. If there no further comments from @ashwinchandrap I will merge this in 2 days.

**Pull request merged**